server/handler/static_files.go

```diff
@@ -33,19 +35,25 @@ type options struct {
 
 // ServeHTTP serves any static file from static directory or fallbacks on index.hml
 func (s *Static) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	filepath := s.dir + r.URL.Path
+	filepath := path.Join(s.dir, r.URL.Path)
 	b, err := assets.Asset(filepath)
 	if err != nil {
+		if strings.HasPrefix(filepath, path.Join(s.dir, "static")) {
```

Comment:
maybe it's better to avoid hard-coded directory name here and put in constant?

Reply:
This file contains other two strings that should be extracted to constants:
- `window.REPLACE_BY_SERVER`
- `index.html`
For consistency, I'll define all the three constants if you agree @smacker

Reply:
that would be great, thanks!

Reply:
done in the rebased/amended commit, thanks